lib/openstudio-standards/standards/Standards.PlanarSurface.rb

```diff
@@ -146,4 +198,73 @@ def planar_surface_apply_standard_construction(planar_surface, climate_zone, pre
 
     return previous_construction_map
   end
+
+  # Get appropriate construction object based on type of surface or subsurface
+  # @author: Doug Maddox, PNNL
+  # @param: surface_category [string] type of surface: this is not an OpenStudio string
+  # @param: surface_type [string] SubSurfaceType: this is an OpenStudio string
+  # @param: cons_set [object] DefaultSubSurfaceConstructions object
+  # @return: [object] Construction object
+  def get_default_surface_cons_from_type(surface_category, surface_type, cons_set)
```

Comment:
this may be a bit nitpicky, but I think 'get_default_surface_cons_from_surface_type' would be a better method name than 'get_default_surface_cons_from_type'.  It's slightly more descriptive.